src/HH4b/processors/bbbbSkimmer.py

```diff
@@ -398,8 +404,8 @@ def process(self, events: ak.Array):
             *selection_args,
         )
 
-        # Txbb pre-selection cut on leading jet
-        # txbb_cut = np.sum(ak8FatJetVars["ak8FatJetPNetXbb"] >= self.preselection["Txbb0"], axis=1)
+        # Txbb pre-selection cut
```

Comment:
did you mean to apply this cut to both jets now?

Reply:
no, I was still going to keep it out, should we apply it?

Reply:
oh I didn't see that it wasn't being applied in the end. I guess probably better to leave it out - I was just confused because that cut looks like it's applied to both jets, whereas I assumed we'd only want it on the leading jet (but I see now it doesn't matter since it's not applied in the end anyway).